src/metabase/analytics/sdk.clj

```diff
@@ -41,7 +41,8 @@
     [embedding-sdk-client :ok]       (prometheus/inc! :metabase-sdk/response-ok)
```

Comment:
i think we should relax this `condp`. Perhaps log if something comes through unrecognized? Right now it will throw an `IllegalArgumentException` and that's heavy handed.

Reply:
I can add some logging there, but note: I've already relaxed the condp by adding `nil` on line 45 which allows it to take any args, and never throw `IllegalArgumentException`. I can add in some logging instead of the `nil` that's acting as the fallthrough condition there, even though I think this shouldn't get hit due to the when-let on line 64.

Reply:
ah! yes you did. I'm sorry i missed that. Ignore what i said. You directly addressed it and i miss it.

Reply:
no worries, I added the logging! 👍